solutions/usaco-991.mdx

```diff
@@ -67,5 +67,40 @@ public class Loan {
 ```
   
 </JavaSection>
+
+<PySection>
+
+```python
+with open('loan.in') as read:
+	n, k, m = map(int, read.readline().split())
+
+def check(mid):
```

Comment:
what does this function do?

Reply:
not really addressed
check if what's possible?
also type annotations

Reply:
check if loan is possible

Reply:
type annotations